app/models/labware_creators/multi_stamp_tubes.rb

```diff
@@ -78,6 +78,27 @@ def request_hash(transfer, child_plate)
       }
     end
 
+    def occupied_wells(wells)
+      wells.each_with_object([]) do |well, occupied_wells|
+        next if well.empty?
+
+        occupied_wells << well
+      end
```

Comment:
This can simply be:
`wells.reject(:empty?)`

Most the useful methods on array actually exist in the Enumerable module. I've actually bookmarked this page, as there's a surprising amount of flexibility there:
https://ruby-doc.org/core-2.7.4/Enumerable.html

Reply:
Much more succinct, thx